streaming/src/main/scala/org/apache/gearpump/streaming/task/TaskActor.scala

```diff
@@ -86,36 +85,13 @@ class TaskActor(val taskContextData : TaskContextData, userConf : UserConfig, va
   def onStop() : Unit = task.onStop()
 
   def output(msg : Message) : Unit = {
-    if (null == outputTaskIds || outputTaskIds.length == 0) {
-      return
-    }
-
-    val partitions = partitioner.getPartitions(msg, taskId.index)
-
-    var start = 0
-
-    sendThroughput.mark(partitions.length)
-
-    while (start < partitions.length) {
-      val partition = partitions(start)
-
-      transport(msg, outputTaskIds(partition))
-      val ackRequest = flowControl.sendMessage(partition)
-      if (null != ackRequest) {
-        transport(ackRequest, outputTaskIds(partition))
-      }
-
-      start = start + 1
-    }
+    this.subscriptions.foreach(_._2.sendMessage(msg))
```

Comment:
Yes. This is a good bypass. I created issue #730 before your comment. 

I am not sure this bypass can work for all cases and languages like Java or not. I will post your bypass in #730.